ui/app/src/views/HomeDashboard.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Box, useTheme } from '@mui/material';
+import {
+  ErrorAlert,
+  ChartsThemeProvider,
+  generateChartsTheme,
+  EChartsTheme,
+  PersesChartsTheme,
+} from '@perses-dev/components';
+import { PluginRegistry, PluginBoundary } from '@perses-dev/plugin-system';
+import { DashboardResource } from '@perses-dev/core';
+import ViewDashboard from '../views/ViewDashboard';
+import { DataSourceRegistry } from '../context/DataSourceRegistry';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import { useBundledPlugins } from '../model/bundled-plugins';
+import { useSampleData } from '../utils/temp-sample-data';
+
+// app specific echarts option overrides, empty since perses uses default
+// https://apache.github.io/echarts-handbook/en/concepts/style/#theme
+const ECHARTS_THEME_OVERRIDES: EChartsTheme = {};
+
+const DEFAULT_DASHBOARD_ID = 'node-exporter-full';
+
+function HomeDashboard() {
```

Comment:
Any thoughts on how to name these components? I like DashboardApp.tsx better but that is used in the dashboards package. Also, it's confusing to me that we have a ViewDashboard.tsx in ui/app and another in ui/dashboards, but not sure of better names / which should be changed?

Reply:
Yeah DashboardApp here makes sense too. Naming is tough lol

Can we combine the two ViewDashboard into one file so all the providers are in one place?

Reply:
Agreed I hate naming things 😆 

So the reason TimeRangeStateProvider is separate and inside the Perses app is since I wanted to use [useSearchParams](https://reactrouter.com/en/6.4.0/hooks/use-search-params) from React Router v6 in handleonTimeRangeChange and I didn't want to make that a required dep in @perses-dev/dashboards (especially since we're on v5 in our UI)

Reply:
oh i see, how about PersesApp.tsx?

Reply:
I wouldn't name it `PersesApp`, just because it's not the home page of Perses which is already called `App.tsx`.

Perhaps we should rename the file in the package `dashboard` and have here `DashboardViews` or `DashboardPage`.

There is a documentation about the naming I think written by @LukeTillman [here](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md). Perhaps it will help :)

Reply:
Ended up changing this back to how it was before since the app component reorg isn't necessary since I switched to the QueryStringProvider approach (TimeRangeStateProvider is back inside dashboards package)